Userland/DevTools/HackStudio/AutoCompleteResponse.h

```diff
@@ -102,4 +103,33 @@ inline bool decode(Decoder& decoder, GUI::AutocompleteProvider::Declaration& dec
     return true;
 }
 
+template<>
+inline bool encode(Encoder& encoder, const Cpp::TodoEntry& entry)
```

Comment:
According to the new conventions, new code should use east consts (Please also correct this in other places in the code).

Reply:
Done :)